libs/core/src/lib/utils/directives/off-screen-element/off-screen-element.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+export type ariaLiveAssertionType = 'polite' | 'assertive' | 'off';
+export type ariaLiveRelevantType = 'additions' | 'removals' | 'text' | 'all';
+
+/**
+ * This directives puts any html element far left of the screen, so it is not visible.
+ * It includes aria-live attribute, so screen reader will be able to read any change in that element.
+ */
+
+@Component({
+  // tslint:disable-next-line:component-selector
+  selector: '[fd-off-screen], [fdOffScreen]',
+  exportAs: 'fdOffScreen',
+  styleUrls: ['./off-screen-element.component.scss'],
+  template: `<ng-content></ng-content>`,
+  encapsulation: ViewEncapsulation.None,
+  host: {
+    class: 'fd-off-screen'
+  }
+})
+export class OffScreenElementComponent {
+  /** set value for aria-live attribute. possible values are 'polite', 'assertive' and 'off'. Default is 'polite' */
+  @Input()
+  @HostBinding('attr.aria-live')
+  ariaLive: ariaLiveAssertionType = 'polite'
+
+  /** set value for aria-atomic attribute. possible values are true and false. Default value is 'true' */
+  @Input()
+  @HostBinding('attr.aria-atomic')
+  ariaAtomic = true
+
+  /** set value for aria-relevant attribute. possible values are 'additions' | 'removals' | 'text' | 'all' */
+  @Input()
+  @HostBinding('attr.aria-relevant')
+  ariaRelevant: ariaLiveRelevantType | ariaLiveRelevantType[];
+
+  /** @hidden */
+  constructor() {}
```

Comment:
if it is not there then some extra typedoc gets generated for constructor by default. It was looking odd, to restrict that, it is needed.

Reply:
<img width="1247" alt="Screenshot 2021-07-07 at 2 23 15 PM" src="https://user-images.githubusercontent.com/57661487/124730419-f19cb200-df2e-11eb-8ba7-66e507d80bad.png">

here i have removed constructor, so above type-doc gets generated.
but with current changes, this does not show.

Reply:
that is strange, because it doesn't appear usually when marked `hidden`. for eg: [card](https://deploy-preview-5864--fundamental-ngx.netlify.app/#/core/card/api) , [facet](https://deploy-preview-5864--fundamental-ngx.netlify.app/#/core/facets/api)

Reply:
i mean, when constructor is removed from code. then above docs get generated.
To avoid that i have used `constructor` and marking it `@hidden`.

Reply:
oh got it, i misunderstood. [this](https://github.com/SAP/fundamental-ngx/pull/5864#discussion_r665177766) is happening in other components also anyway(Info label), so may not be a problem, but I'll mark this as resolved.